diagnostic.md

```diff
@@ -32,7 +32,20 @@ format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
 <!-- your answer starts here -->
-
+html>
+    <head>
+      <title>My Page Asslym</title>
+    </head>
+    <body>
+      <div class="unordered=list">
+      <ul>
+        <li>Coffee</li>
+        <li>Tea</li>
+        <li>Milk</li>
+      </ul>
+    </div>
+    </body>
+</html
```

Comment:
Missing a less than symbol at the beginning of your HTML tag and a greater than symbol at the end of your closing HTML tag. ie:
`+<html>
 +    <head>
 +      <title>My Page Asslym</title>
 +    </head>
 +    <body>
 +      <div class="unordered=list">
 +      <ul>
 +        <li>Coffee</li>
 +        <li>Tea</li>
 +        <li>Milk</li>
 +      </ul>
 +      </div>
 +    </body>
 +</html>
`